controller/experiment-controller.go

```diff
@@ -79,7 +79,12 @@ func (ct *Controller) InjectExperiment(ctx *gin.Context, kind string, chaos chao
 	chaosNamespace := chaos.GetNamespace()
 	experiment, err := ct.ChaosClient.GetExperiment(ctx, chaosNamespace, chaosName, kind)
 	if err != nil {
-		exists = false
+		if err == chaosmesh.NotFoundErr {
```

Comment:
I prefer to use `errors.Is` instead of comparing directly, `errors.Is` could resolve the wrapper of the error.

Reply:
how about `errors.IsNotFound(err)`, I update this PR, PTAL @STRRL